docker-compose.yml

```diff
@@ -14,5 +14,15 @@ services:
       - SPRING_PROFILES_ACTIVE=dev
       - SPRING_JPA_PROPERTIES_HIBERNATE_SHOW_SQL=false
 
+  postgres:
+    image: postgres:13
```

Comment:
_suggestion_
`postgres:13-alpine` is about 50% smaller

```
$ docker images | grep postgres
postgres   13                      f51c55ac75ed        12 days ago         314MB
postgres   13-alpine               770f846cacce        2 weeks ago         159MB
```

Reply:
👍 thanks for this - i will update